common/views/pages/_app.tsx

```diff
@@ -83,8 +83,9 @@ const WecoApp: FunctionComponent<WecoAppProps> = ({
 
   const serverData = isServerDataSet ? pageProps.serverData : defaultServerData;
 
-  const onAnalyticsConsentChange = () =>
-    console.log('Decide on what to do here and how to handle');
+  const onAnalyticsConsentChange = () => {
+    // Have popup that says "Your cookie settings have been saved"
```

Comment:
As in [mockups](https://www.figma.com/file/N76hkJsMkSb2O6ToyQPxax/Cookie-journeys-update?node-id=1718%3A84945&mode=dev), but review with Jason.

Reply:
Remove this